examples/Microsoft.Spark.CSharp.Examples/Sql/Batch/readmes/LoggingReadme.md

```diff
@@ -0,0 +1,83 @@
+# Log Processing with Batch Data
+
+In this sample, you'll see how to use [.NET for Apache Spark](https://dotnet.microsoft.com/apps/data/spark) 
+to analyze a log file. In the world of big data, this is known as **log processing**.
+
+## Problem
+
+Our goal here is to determine if strings in a file are complete log entries and to then analyze their contents. We'll be analyzing the 
+[Apache Unix Log Samples](http://www.monitorware.com/en/logsamples/apache.php). The zip file contains both access logs and error logs, 
```

Comment:
This link doesn't work for me.

Reply:
Also in the GitHub examples you have a Dataset section.  Do you want to move this and the following line to that section ?

Reply:
Looks like it got taken down, I'll update to a new link/log file resource. And good idea about Dataset section for consistency, I'll add that too,